buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -127,14 +127,16 @@ def print_projects_need_to_migrate(failed_jobs_per_flag):
 
 
 def print_flags_need_to_migrate(failed_jobs_per_flag):
-    info_text = ["#### Downstream projects need to migrate for the following flags"]
-    for flag, jobs in failed_jobs_per_flag.items():
+    for flag, jobs in reversed(list(failed_jobs_per_flag.items())):
```

Comment:
Why reversed()? Also, we need to sort the flags first.

Reply:
I added a comment about how using reverse. And good catch, I thought the flags were sorted.